### PR TITLE
updated PULL-REQUEST-TEMPLATE.md

### LINKED ISSUE #672: Update Pull Request template
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**: Replace `@publiclab/is-reviewers` by `@publiclab/reviewers and @publiclab/is-reviewers`.

**code link** : https://github.com/publiclab/image-sequencer/blob/main/.github/PULL_REQUEST_TEMPLATE.md

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by harsh1599**
Fixes #672 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@harsh1599 welcome to the image sequencer community :tada:. Thanks for your PR. You can mention @publiclab/is-reviewers to ask for a review in PRs also you can replace the `#[add issue number here.]` with a link to the issue page. You can also check all of the checkboxes to make it easier for the reviewers to review your pr. Thanks!

**Comment by harsh1599** [comment]
I updated the comment as you said..is there anything else to the process?

**Comment by HarshKhandeparkar** [comment]
No you are good to go. Let the reviewers review this

**Comment by harsh1599** [comment]
@HarshKhandeparkar  okay! Thanks for the help and instant replies

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Make sure these boxes are checked before your pull request (PR) is ready to be r
 * [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
 * [ ] code is in uniquely-named feature branch and has no merge conflicts
 * [ ] PR is descriptively titled
-* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
+* [ ] ask `@publiclab/reviewers and @publiclab/is-reviewers` for help, in a comment below
```
Comment (gauravano) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below
```

Comment (HarshKhandeparkar) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please commit the changes suggested.

**Comment by harsh1599** [comment]
I have made the changes

**Comment by jywarren** [comment]
Great work, thanks!!!!

**Pull request merged**